package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "s47_list_of_inspirational_movies",
+  "version": "1.0.0",
+  "description": "This is your first repository",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
```

Comment:
Consider adding more information to the package.json file, such as the author's name, relevant keywords, and any dependencies or scripts that the project uses. [medium]